package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "manageiq-ui-classic",
   "version": "1.0.0",
-  "description": "ManageIQ Cloud Management Platform",
+  "description": "ManageIQ Cloud Management Platform ",
```

Comment:
This change shouldn't be here.

Reply:
yes, there was a test PR (with a simple change) to check if forking and merging work correctly between 2 users. will change it back soon.